Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM node:12.5.0
```

Comment:
is this part of the vulnerability fix?

Reply:
I'm bringing it up to the later version we just updated Persona, so that devs don't have to switch back and forth when debugging multiple services.